examples/basic_tutorial_BERT.ipynb

```diff
@@ -0,0 +1,1198 @@
+{
```

Comment:
<p>if these two cells are "almost" duplicated, can you please create a function in one cell and call it with different parameters?</p><p> </p><p>Probably same comments may work a few cells above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-deep/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>